pkgs/development/web/woff2/brotli-dependencies.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ecfbb83..da8ecf1 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -21,16 +21,6 @@ option(NOISY_LOGGING "Noisy logging" ON)
+ # Version information
+ set(WOFF2_VERSION 1.0.2)
+ 
+-# When building shared libraries it is important to set the correct rpath
+-# See https://cmake.org/Wiki/CMake_RPATH_handling#Always_full_RPATH
+-set(CMAKE_SKIP_BUILD_RPATH FALSE)
+-set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+-list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_LIBDIR}" isSystemDir)
+-if ("${isSystemDir}" STREQUAL "-1")
+-  set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_LIBDIR}")
+-endif()
```

Comment:
Can we do without this part? Maybe, `cmakeFlags = "-DCMAKE_SKIP_BUILD_RPATH:BOOL=TRUE"`?